### PR TITLE
Release 0.20

### PR DESCRIPTION
**Pull request by CyberHoward**
Tracking branch for `v0.20`

### Checklist

- [x] CI is green.
- [x] Changelog updated.

### CODE REVIEW

File: framework/Cargo.toml
Comment (Buckram123) [inline framework/Cargo.toml:1]: Don't forget to include `Cargo.lock` files so wasms will remain reproducible

Comment (Buckram123) [reply framework/Cargo.toml:1]: @CyberHoward so what do you think? Want it remain compilable in the future

Comment (CyberHoward) [reply framework/Cargo.toml:1]: Yeah you're right, should include lock file

**Review by Buckram123 — APPROVED** [review]
LGTM

**Pull request merged**